pom.xml

```diff
@@ -87,6 +87,7 @@
 		<tomcat.version>9.0.43</tomcat.version>
 		<ical4j.version>3.0.18</ical4j.version>
 		<cxf.version>3.4.2</cxf.version>
+		<io.prometheus.version>0.10.0</io.prometheus.version>
```

Comment:
Prometheus Java Client and all new dependencies are Apache Licence 2.0: 
https://github.com/prometheus/client_java/blob/master/LICENSE